frontend/src/views/AuthPage.vue

```diff
@@ -0,0 +1,184 @@
+<script>
+import authService from '../api/auth';
+import { ref } from 'vue';
+import { useRouter } from 'vue-router';
+import { useUserStore } from '@/stores/user';
+
+export default {
+  name: 'auth-page',
+  data: () => {
+    return { isToggled: false };
+  },
+  setup() {
+    const router = useRouter();
+    const userStore = useUserStore();
+
+    const loginForm = ref({
```

Comment:
Nitpick: as far as I can tell, the `loginForm` and `registerForm` objects do not get fully replaced at any point. So the `reactive` method can be used instead of `ref` and accessing through the `value` prop can be avoided.